### ISSUE TITLE
Add ur5 model (and replace Romeo)

### ISSUE THREAD

**Issue by nmansard** [stack-of-tasks/pinocchio#546]
For the examples of the documentation, it is kind of necessary to have easy access to some urdf models. We already have Romeo, which is good. It would be nice to have a manipulator model. I suggest to include UR5, but any alternative is welcome. The doc yet proposes to use this repo:
https://github.com/humanoid-path-planner/universal_robot
However, this repo does not have a plain urdf model but rather some xacro template model. Could we have another repo providing a plain urdf model of a manipulator?

About Romeo, should we remove Romeo and add Talos instead, to follow the scientific work of Gepetto?

**Comment by nim65s** [comment]
I agree, and if no one disagree, I will do this.

**Comment by jcarpent** [comment]
We must keep romeo for the unit tests at least.

**Comment by jcarpent** [comment]
https://github.com/humanoid-path-planner/ur_description is correct

**Comment by olivier-stasse** [comment]
The ros way is to do:
rosrun xacro xacro --inorder -o model.urdf model.urdf.xacro
This generates the urdf file.

**Comment by nmansard** [comment]
Ok: let's do: keep romeo ; add the UR5 repo proposed by Justin and containing the .urdf file. 
I also suggest to add Talos repo and implement future humanoid tests with Talos and not Romeo.

**Comment by cmastalli** [comment]
We might want to create a robot data repository. This doesn't intend to replace real robot models, it just for algorithm benchmarking (not only for Pinocchio).

**Comment by jcarpent** [comment]
This indeed a good idea to collect all the models in a single repository. I already found such a repository on github, but I'm not able to remind where exactly.

**Comment by jcarpent** [comment]
Before creating something from scratch, can you check if a similar repository does not already exist?

**Comment by cmastalli** [comment]
I didn't find a such kind of repository (15 min searching on github). The only database, that I know, are inside bullet3 and dartsim. They're not sub-modules and not necessarily interesting robots for us.

**Comment by nim65s** [comment]
I have made https://github.com/stack-of-tasks/pinocchio-benchmarks/tree/master/models, but only with urdf. I think we also need the meshes

**Comment by jcarpent** [comment]
Indeed, we need meshed too, and srdf!

**Comment by nim65s** [comment]
I will extract this folder from pinocchio-benchmarks to create a new repo. I was also thinking about something like robots/robot_db.

**Comment by jcarpent** [comment]
I think you can keep the robot_description organization with urdf, meshes and srdf folders.

**Comment by nmansard** [comment]
Could we make a decision on this issue? I would need an answer for the Pinocchio class at the end of the month.

**Comment by nim65s** [comment]
I have been working on this recently, and this is now OK for romeo-description in robotpkg, and I am almost ready for universal-robot-description.

@stonneau & @pFernbach added https://github.com/humanoid-path-planner/hpp-rbprm-robot-data with HRP-2, HyQ & Talos, and I will add this in robotpkg soon, so this can also be useful

**Comment by nmansard** [comment]
After another oral discussion with @nim65s and @olivier-stasse , here is a proposition of solution:
- @nim65s will create new git packages with hard-coded urdf files (i.e. no xacros, no pre-treatment needed) for a selection of robots of used by gepetto friends.
- @nim65s will also create a git meta-package gathering all these models using submodules.
- finally, @nim65s (again) will package this meta-package in robotpkg in two different versions: one with the meshes, and one without the meshes.
Consequently, tutorials and new users can use directly the model with meshes using git clone --recursive. Unittest and benchmarks can be performed using the binary packages without the meshes.

We also mentioned that the current talos model is under proprietary license. @olivier-stasse will see with PAL if we can change it. 

We also discussed the possible danger in having an less-up-to-date version of a xacro-based model (like the talos model that sometime evolves). Attention should be paid so that there is no possible confusion for an distracted user. I proposed to change a detail of the urdf-only models (like changing the color of an important body, or adding a frame with an evident name like "this-urdf-should-be-used-in-tutorials-only". @olivier-stasse rather proposed to strengthen the automatization of the parsing in robot application like StackOfTasks.
